libs/platform/ebpf_hash_table.c

```diff
@@ -51,6 +51,7 @@ struct _ebpf_hash_table
 {
     uint32_t bucket_count;          // Count of buckets.
     volatile int32_t entry_count;   // Count of entries in the hash table.
+    int32_t max_entry_count;        // Maximum number of entries allowed or EBPF_HASH_TABLE_NO_LIMIT if no maximum.
```

Comment:
Linux appears to allow 2^32 entries.  It looks like we only allow 2^31 entries.
Would be better to make these uint32_t since you cannot actually have a negative number of entries.